sdks/python/apache_beam/io/bigquery.py

```diff
@@ -335,6 +336,24 @@ def __init__(self, table=None, dataset=None, project=None, query=None,
     self.validate = validate
     self.coder = coder or RowAsDictJsonCoder()
 
+  def display_data(self):
+    res = {}
+    if self.query is not None:
+      res['query'] = DisplayDataItem(self.query, label='Query')
```

Comment:
What's the difference between label and key? Will users almost always have both? Do we need both?